### DISCUSSION TITLE
Why deepcopy the scheduler in script train_dreambooth_lora_sd3.py

### DISCUSSION THREAD

**Discussion by gitlabspy** [huggingface/diffusers#8818] [Q&A]
Can anyone tell me why the scheduler can not be used directly?

https://github.com/huggingface/diffusers/blob/a785992c1d6fcb1ff66f8a0d68d09a0a81b909e8/examples/dreambooth/train_dreambooth_lora_sd3.py#L1150

**Comment by ShoufaChen** [comment]
Also here
https://github.com/huggingface/diffusers/blob/86bcbc389e810c3d12a61dce57dd02071c6b3050/examples/dreambooth/train_dreambooth_flux.py#L1112

@linoytsaban, could you please give some hints about this operation? thanks in advance.

**Comment by Qabrix** [comment]
Any answers to this question?

**Comment by Longin-Yu** [comment]
I have the same question. Thanks if someone could answer to this.
